### ISSUE TITLE
Add recommended labels

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#5]
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

**Comment by dwbrown2** [comment]
@luisdavim I have #24 right now. Anything else you would want to see there?

**Comment by sebastien-prudhomme** [comment]
@dwbrown2 the recommended labels fro Helm charts are there: https://helm.sh/docs/chart_best_practices/#labels-and-annotations

In practice for an application with only one component (without taken into account subcharts)
```
  labels:
    app.kubernetes.io/name: {{ include "cost-analyzer.name" . }}
    helm.sh/chart: {{ include "cost-analyzer.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }
```

This must be added in every resource managed by the chart (cronjob, service, service account, ...)

The easiest way to do that is to add a ```define``` in the helper YAML file with all this labels and use that ```define``` in every resource

**Comment by dwbrown2** [comment]
Thanks for sharing @sebastien-prudhomme! There seems to be a small amount of conflict between helm and kubernetes recommendations (e.g. helm vs tiller for managed-by). I don't have a strong opinion here but just an observation. 

The only label that I worry about is `app.kubernetes.io/instance`. We've seen teams install multiple deployments of Kubecost and it's not clear to me they would know these need to be unique. Let me give this some thought. Also, I would welcome your opinion on this!

**Comment by sebastien-prudhomme** [comment]
@dwbrown2 with the current version of Helm (v2), the release name must be unique in a Kubernete cluster.

With the next version (v3), the release name will be namespace scoped.

**Comment by luisdavim** [comment]
There is no conflict.  Helm recommends the same labels as kubernetes. If you run helm create with a recent version of helm those are the labels that helm will set by default in all templates.

**Comment by luisdavim** [comment]
Also the label that you mentioned doesn't have to be unique, the combination of labels should be.
I'd suggest getting the latest stable release of helm and using helm create to check what labels and selectors are used.

**Comment by dwbrown2** [comment]
Fixed by PR #58 from Sébastien!
